src/TBuffer.h

```diff
@@ -155,7 +155,7 @@ class TBuffer
     QString bufferToHtml(const bool showTimeStamp = false, const int row = -1, const int endColumn = -1, const int startColumn = 0,  int spacePadding = 0);
     int size() { return static_cast<int>(buffer.size()); }
     bool isEmpty() const { return buffer.size() == 0; }
-    QString& line(int n);
+    QString& line(int lineNumber);
```

Comment:
Since the argument name is trivial to deduce it can just as easily be omitted I think.
```suggestion
    QString& line(int);
```

Reply:
Personally, I'm not sure if it adds much value because it might create a bit of cognitive load for developers who now need to deduce it - personally, doing this doesn't seem to add any value and to me this is a C++ antifeature that is to be avoided.